scrabble.js

```diff
@@ -1,8 +1,107 @@
 var Scrabble = function() {};
 
-// YOUR CODE HERE
-Scrabble.prototype.helloWorld = function() {
-  return 'hello world!';
+// setup score key
+Scrabble.prototype.score_key = {
+  1:  ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
+  2:  ["D", "G"],
+  3:  ["B", "C", "M", "P"],
+  4:  ["F", "H", "V", "W", "Y"],
+  5:  ["K"],
+  8:  ["J", "X"],
+  10: ["Q", "Z"]
 };
 
+Scrabble.prototype.getKey = function(value) {
+  var value = value.toUpperCase();
+  for(var key in this.score_key) {
+    if (this.score_key[key].join().includes(value)) {
+      return Number(key);
+    }
+  }
+
+  return null;
+}
+
+// actually do stuff now
```

Comment:
You've probably already had it pointed out that any unnecessary comments should be removed before submitting your final code for review.   Comments are always a good thing, though, when they highlight a piece of code that needs some description.